kraken-std/pyproject.toml

```diff
@@ -39,6 +39,7 @@ isort = "*"
 mypy = "*"
 pycln = "*"
 pytest = "*"
+pytest-cov = "*"
```

Comment:
We only need this here if we actually make use of it in the build script for this project.

Reply:
Yes I will add that